### DISCUSSION TITLE
Consider bake in camel-snake-kebab

### DISCUSSION THREAD

**Discussion by sg-qwt** [babashka/babashka#1595] [Ideas]
camel-snake-kebab is lightweight and handy for scripting. And personally I've been using that as often as cheshire. 

Hope it's a good candidate to be builtin.

**Comment by borkdude** [comment]
It's funny, I almost never use this library. Maybe because I just work with the keys as is and don't experience it as a problem.
This library does already work with babashka when you include it in bb.edn etc.
